openeo_processes_dask/process_implementations/cubes/resample.py

```diff
@@ -0,0 +1,79 @@
+import logging
+from typing import Union
+
+import odc.geo
+from odc.geo.geobox import resolution_from_affine
+from pyproj.crs import CRS, CRSError
+
+from openeo_processes_dask.process_implementations.data_model import RasterCube
+
+logger = logging.getLogger(__name__)
+
+resample_methods_list = [
+    "near",
+    "bilinear",
+    "cubic",
+    "cubicspline",
+    "lanczos",
+    "average",
+    "mode",
+    "max",
+    "min",
+    "med",
+    "q1",
+    "q3",
+]
+
+
+def resample_spatial(
+    data: RasterCube,
+    projection: Union[str, int] = None,
+    resolution: Union[str, int] = None,
+    method: str = "near",
+    align: str = "upper-left",
+):
+    """Resamples the spatial dimensions (x,y) of the data cube to a specified resolution and/or warps the data cube to the target projection. At least resolution or projection must be specified."""
+
+    if align == "upper-left":
```

Comment:
1) This should especially appear if the user chose a different option than the default
2) If we don't support it, shouldn't we just remove it from the spec we expose?

Reply:
Yeah happy to remove it!